app/controllers/list_controller.rb

```diff
@@ -0,0 +1,43 @@
+class ListController < ApplicationController
+  def index
+    @lists = List.all
+  end
+
+  def edit
+    @list = List.find_by_id(params[:id])
```

Comment:
Will remove this (as well as the edit spec) since I don't think we have a mockup for the edit page yet